sources/packages/forms/src/form-definitions/confirmsstudentenrollment.json

```diff
@@ -412,6 +412,595 @@
           "input": false,
           "tableView": false,
           "components": [
+            {
+              "label": "HTML",
+              "labelWidth": "",
+              "labelMargin": "",
+              "tag": "h2",
+              "className": "category-header-large primary-color",
+              "attrs": [
+                {
+                  "attr": "",
+                  "value": ""
+                }
+              ],
+              "content": "Disability Information",
+              "refreshOnChange": false,
```

Comment:
We use to enable `refreshOnChange` to true on HTML components using form data values.

With the current editor, we don't have the option to change it from UI. But I believe we must enable from definition. 

![image](https://github.com/bcgov/SIMS/assets/54600590/9a91efa7-9fad-4749-8019-b171a29061f2)